### ISSUE TITLE
If we do not use neural network, how many logloss can we achieve?

### ISSUE THREAD

**Issue by guotong1988** [YuriyGuts/kaggle-quora-question-pairs#4]
and how many dim of features for the machine learning model without neural network?
Thank you.
@YuriyGuts

**Comment by YuriyGuts** [comment]
You can see the number of features in each feature list in the output of the [final model notebook](https://github.com/YuriyGuts/kaggle-quora-question-pairs/blob/master/notebooks/classify-lightgbm-cv-pred.ipynb). So that would be around 190. As for the logloss question, I haven't tried that. I'd probably expect around 0.20-0.25?
